From bfa2e99d1ceec6d24e32127643d04372b9b9fd6b Mon Sep 17 00:00:00 2001
From: Ilya Gorchakov <ilya.gorchakov@adakta.ru>
Date: Tue, 30 Jan 2018 18:43:26 +0300
Subject: [PATCH] Add adakta nano6ull support

diff --git a/arch/arm/mach-imx/mach-imx6ul.c b/arch/arm/mach-imx/mach-imx6ul.c
index 034db69..7c576b2 100644
--- a/arch/arm/mach-imx/mach-imx6ul.c
+++ b/arch/arm/mach-imx/mach-imx6ul.c
@@ -50,10 +50,13 @@ static int ksz8081_phy_fixup(struct phy_device *dev)
 	return 0;
 }
 
-#define PHY_ID_KSZ8081	0x00221560
+#define PHY_ID_KSZ8081_REV0	0x00221560
+#define PHY_ID_KSZ8081_REV1	0x00221561
+
 static void __init imx6ul_enet_phy_init(void)
 {
-	phy_register_fixup_for_uid(PHY_ID_KSZ8081, 0xffffffff,	ksz8081_phy_fixup);
+	phy_register_fixup_for_uid(PHY_ID_KSZ8081_REV0, 0xffffffff,	ksz8081_phy_fixup);
+	phy_register_fixup_for_uid(PHY_ID_KSZ8081_REV1, 0xffffffff,	ksz8081_phy_fixup);
 }
 
 #define OCOTP_CFG3			0x440
-- 
1.9.1

